var/spack/repos/builtin/packages/suite-sparse/package.py

```diff
@@ -35,7 +35,7 @@ class SuiteSparse(Package):
     version('4.5.3', '8ec57324585df3c6483ad7f556afccbd')
     version('4.5.1', 'f0ea9aad8d2d1ffec66a5b6bfeff5319')
 
-    variant('tbb', default=True, description='Build with Intel TBB')
```

Comment:
@eklee15 according to https://software.intel.com/en-us/forums/intel-threading-building-blocks/topic/292622 it should work. Do you have problems building TBB on your system?